lib/InputDeployerVerify.js

```diff
@@ -67,35 +77,115 @@ async verifyBeforeDeployment() {
               }
            }
        }
+  if(prevEnv){
+       var mapRfdPrevEnvIssue = new Map()
+       for (var issue of rfdIssueLinks){
+         var rfdInfo = await jiraClient.getIssue(issue)
+         if(rfdInfo.fields.customfield_10121.value.toLowerCase()===prevEnv){
+             mapRfdPrevEnvIssue.set(issue,rfdInfo.fields.status.name)
+         }    
+      }
+   }
+
+
```

Comment:
The code seems to be getting quite long for more than 200 lines for the condition block. Is it possible to separate it into smaller chunks or logics or leave some comments to explain the logic before the long block might be helpful.